.gitignore

```
@@ -4,6 +4,9 @@ lib/
 # ML Models
 models/
 
+# API keys
+businextai-f686af52ed49.json
```

This seems oddly specific :thinking: 

```suggestion
businextai-[0-9a-f]*.json
```